arm-storsimple8000series/2017-06-01/swagger/storsimple.json

```diff
@@ -1745,6 +1996,31 @@
         ],
         "parameters": [
           {
+            "name": "deviceName",
```

Comment:
if you're defining this parameter multiple times for usage in multiple operations, you could move its definition to the global parameters section.